docs/lorawan-on-helium/proof-of-coverage.mdx

```diff
@@ -51,9 +51,9 @@ tabled because they were deemed too complex to implement will become viable and
 # Step 1: Oracle-Based PoC - In Progress
 
 From the launch of the network in 2019 until February 2, 2023, the IOT PoC system relied on a
-“Challenger, Beaconer, Witness, Validator, Rewarder” model. In this model at least five separate
-actors had to work together in order to make PoC events happen. Any break in the chain meant the
-loss of the entire event and its rewards.
+“Challenger, Beaconer, Witness, Rewarder” model. In this model at least five separate actors had to
```

Comment:
This paragraph is historical in context. Can revert this change.

Reply:
reverted